assets_manifest.json

```diff
@@ -2,11 +2,11 @@
   "data_assets": [
     {
       "id": "smbertVocab",
-      "url_suffix": "smbert_v0000/vocab.txt"
+      "url_suffix": "smbert_v0001/vocab.txt"
     },
     {
       "id": "smbertModel",
-      "url_suffix": "smbert_v0000/smbert.onnx",
+      "url_suffix": "smbert_v0001/smbert-quantized.onnx",
       "chunk_size": "11MB"
```

Comment:
or we can keept the `chunk_size` and only upload the model without the fragments

Reply:
won't this change break `xayn_ai`?

Reply:
both ways should be fine. but since still have the old xayn_ai, we should not remove the `chunk_size` and only upload the model as a whole. (so that we have in the end the model as a whole and splited into fragments